packages/@okta/vuepress-site/docs/guides/authenticators-okta-verify/main/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Okta Verify (Push/OTP) integration guide
+---
+
+<div class="oie-embedded-sdk">
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+<StackSelector />
+
+This guide shows you how to integrate Okta Verify into your app using the embedded SDK.
+
+---
+**Learning outcomes**
+
+* Understand the Okta Verify enrollment and challenge flows
+* Learn step-by-step how to integrate Okta Verify into your authentication use case
+
+**What you need**
+
+* <StackSnippet snippet="whatyouneedsdk" />
+* <StackSnippet snippet="whatyouneedorg" />
+
+**Sample code**
+
+* <StackSnippet snippet="samplecode" />
+
+**Related use cases**
+
+<StackSnippet snippet="relatedusecases" />
+
+---
+
+## Overview
+
+...
+
+<!--
+<div class="common-image-format">
+
+![Diagram showing the WebAuthn enrollment flow](/img/authenticators/authenticators-webauthn-flow-overview.png)
+
+</div>
+
+As the service provider, you can provide WebAuthn support to your users by enabling it in your Okta org and building out support for it in your application using the Embedding SDK.
+-->
+
+## Update configurations
+<!--
+
+Before you can start using Web Authentication (WebAuthn), create an Okta org application as described in <StackSnippet snippet="orgconfigurepwdonly" inline/>. Then add WebAuthn to your app integration by executing the following steps:
+
+### Add WebAuthn to the Okta org
+
+1. In the Admin Console, go to **Security > Authenticators**.
+1. On the **Authenticators** page, click **Add Authenticator**.
+1. On the **Add Authenticator** dialog, click **Add** under **FIDO2 (WebAuthn)**.
+1. Leave the default value for **User Verification**, which is set to "Discouraged".
+1. On the **Add FIDO2(WebAuthn)** dialog, click **Add**.
+1. On the **Authenticators** page, select the **Enrollment** tab.
+1. On the **Enrollment** tab, click **Edit** for the **Default Policy**.
+1. Set **FIDO2 (WebAuthn)** to **Optional** and click **Update Policy**.
+
+### Configure your Okta org application to use WebAuthn
+
+1. In the Admin Console, go to **Applications** and **Applications**.
+1. On the **Applications** page, click on the application you've previously created.
+1. On the **General** tab ensure that **Interaction Code** and **Refresh Token** are selected.
```

Comment:
Is that specific enough? It's not just the General tab, it's the General Settings pane of that tab, and it requires editing that pane which is somewhat unintuitive. The simplest thing in cases like this is to tell the developer to select those items... that way if the items are unchecked they get checked. That would be something like:

1. Click **Edit** in the **General Settings** pane.
1. Select **Interaction Code** and **Refresh Token** in the **Application** section.
1. Click **Save** at the bottom of the **General Settings** section.